src/components/popover/Popover.js

```diff
@@ -164,6 +168,7 @@ Popover.propTypes = {
 Popover.defaultProps = {
   active: true,
   backdrop: 'dark',
+  depth: 300,
```

Comment:
Any reason why you're not just calling it zIndex?

Reply:
I was inspired by [teamleader.design](https://teamleader.design/primitives/depths). Although it's a good remark.

Do you guys think I should rename it to `zIndex` instead of `depth`?

Reply:
I think it's better to just use zIndex to avoid any confusion